Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Configs/DriverIntelligentPoolConfig.hs

```diff
@@ -0,0 +1,45 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Domain.Types.Configs.DriverIntelligentPoolConfig where
```

Comment:
All these Configs are parts of Merchant (which is basically config as well), so i think you should move these configs from  `....Configs.DriverIntelligentPoolConfig` to `....Merchant.Configs.DriverIntelligentPoolConfig` (or just `....Merchant.DriverIntelligentPoolConfig` maybe)

Reply:
Good Idea. @0utkarsh Lets Just have this all inside Merchant folder like this `....Merchant.DriverIntelligentPoolConfig`

Reply:
Static BPP part is still not refactored the same way